lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1187,6 +1187,22 @@ def jsonrpc_version(self):
         log.info("Get version info: " + json.dumps(msg))
         return self._render_response(msg, OK_CODE)
 
+    def jsonrpc_report_bug(self, p):
+        """
+        Report a bug to slack
+
+        Args:
+            'message': string, message to send
+        Returns:
+            True if successful
+        """
+
+        bug_message = p['message']
+        platform_name = self._get_platform()['platform']
+        d = defer.succeed(report_bug_to_slack(bug_message, self.lbryid, platform_name, lbrynet_version))
+        d.addCallback(lambda _: self._render_response(True, OK_CODE))
+        return d
```

Comment:
are deferred's necessary here?

I think this can be:

```
report_bug_to_slack(bug_message, self.lbryid, platform_name, lbrynet_version)
return self._render_response(True, OK_CODE)
```